### PR TITLE
Module 1: details popup window

### PR DESCRIPTION
**Pull request by boniface193**
### popup window that includes the project details in both desktop and mobile

 - When the user clicks (or taps) the button to check project details, the popup with details about the project appears
 - When the user clicks (or taps) the close (X) button, the popup disappears.
 - initial project was refactor to use object to store project details

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -1,11 +1,6 @@
 {
-  "extends": [
-    "stylelint-config-standard"
-  ],
-  "plugins": [
-    "stylelint-scss",
-    "stylelint-csstree-validator"
-  ],
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
```
Comment (emmiiorji) [inline .stylelintrc.json:3]: Kindly avoid modifying the linter configurations, they represent coding standards you share with your team.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @boniface193, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Invalid

You have submitted a project that modifies linters configuration. Please ensure that you use the exact configuration for a [HTML/CSS/JS](https://github.com/microverseinc/linters-config/tree/master/html-css-js) project as given my Microverse. Confirm that the `.eslintrc.json`, `.stylelintrc.json`, `.hintrc` and `.github/workflows/linters.yml` files match the ones in your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .stylelintrc.json (hunk 1 above)
Comment (boniface193) [reply .stylelintrc.json:3]: I understand what you mean (after carefully going through the code I realized the configuration was still the same)

**Review by Gambit142 — COMMENTED** [review]
Hi @boniface193 @oyhpnayiaw 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly add the following sections to your _**README.md**_ file:

- The **_Key Features_** Section. This section should contain 1 - 3 features that was implemented in this project
- The **_Future Features_** Section. This section should contain 1 - 3 features that can be implemented in the future 
- The `MIT` link in the file leads me to a 404 error page. Kindly fix this and personalize the LICENSE that you will use. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by boniface193** [comment]
all reame file changes are completed @Gambit142

File: MIT.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [IKechukwu]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
```
Comment (rustam-tolipov) [inline MIT.md:4]: [OPTIONAL] I thing this lines of the MIT license is not necessary. You can remove them.

File: style.css [hunk 3]
```diff
@@ -494,6 +613,15 @@ nav ul li img {
 
 /* medium screen resolution */
 @media (min-width: 767px) and (max-width: 1000px) {
+  .grid-container {
```
Comment (rustam-tolipov) [inline style.css:616]: Your desktop design is excellent. However, there are discrepancies in its mobile design which require attention. Please refer to the attached video:

Solarized dark             |  Solarized Ocean
:-------------------------:|:-------------------------:
mobile   |  Template 1-Mobile-Detail

- [x] 1. If I hower your mobile (modal menu) it breaking the design. 
- [x] 2. Your text is a bit longer than the original design. I suggest you to limit it or you can use overflow method. 
- [x] 3. Your stacks (html, css, javaScript) are alse really close to the description.
- [x] 4. As you can see in the video your buttons are also looks different.

Kindly fix all the mentioned changes 😉

File: style.css [hunk 4]
```diff
@@ -690,18 +818,22 @@ nav ul li img {
...
+    grid-template-columns: auto;
+  }
+
   .card {
-    display: flex;
     flex-direction: column;
     padding: 16px 16px 24px 16px;
-    height: min-content;
+
+    /* height: min-content; */
```
Comment (rustam-tolipov) [inline style.css:829]: Please ensure that any unused commented codes are removed from the file. Thank you.

File: js/projects.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+        const skipFirstChild = main.childNodes;
+        skipFirstChild[2].innerHTML += createDivContainer;
+
+        document
+          .querySelector('#closeProject')
+          .addEventListener('click', () => {
+            createPopupCard.remove();
+            const getLastChild = Array.from(skipFirstChild);
+            getLastChild[1].lastElementChild.remove();
+            window.reload();
```
Comment (rustam-tolipov) [inline js/projects.js:84]: - [x] If I open one of your modal menus, close it, and then attempt to open another modal menu, it does not behave as expected. This indicates that your clothing functions require attention. In the browser object model (BOM) there is no reload() functionality. The reload() function can be found in the location object. To access it, call window, then location and finally reload(). The full form is window.location.reload(), which should solve your issue when implemented correctly.

**Review by rustam-tolipov — COMMENTED** [review]
Hello @boniface193,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

**You are totally free to remove this lines in your Readme:**

_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**